transformer_engine/common/fused_attn/fused_attn_f16_arbitrary_seqlen.cu

```diff
@@ -224,11 +221,21 @@ void fused_attn_arbitrary_seqlen_fwd_impl(
                 std::make_tuple(nullptr), key_tensors_tuple,
                 Stats_tuple, bias_tuple, padding_tuple, dropout_tuple);
 
-            mha_graph->validate();
-            mha_graph->build_operation_graph(handle);
-            mha_graph->create_execution_plans({fe::HeurMode_t::A});
-            mha_graph->check_support(handle);
-            mha_graph->build_plans(handle);
+            if (!mha_graph->validate().is_good()) {
+                NVTE_ERROR("MHA Graph (fwd) validation is unsuccessful.");
```

Comment:
Can we instead introduce something similar to https://github.com/NVIDIA/TransformerEngine/blob/main/transformer_engine/common/util/logging.h#L54?

Reply:
Done.